tests/tests_bond_nm.yml

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# This file was generated by ensure_provider_tests.py
+---
+# set network provider and gather facts
+- hosts: all
+  name: Run playbook 'playbooks/tests_bond.yml' with nm as provider
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+      tags:
+        - always
+
+
+# workaround for: https://github.com/ansible/ansible/issues/27973
+# There is no way in Ansible to abort a playbook hosts with specific OS
+# releases Therefore we include the playbook with the tests only if the hosts
+# would support it.
```

Comment:
FYI, Ansible 2.7 is not maintained anymore (https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#ansible-release-cycle) and since 2.8 there is a way to abort a host in the play. Of course, this code is found in many test playbooks so you may choose to keep it consistent and change it globally at some point.